src/ordered_enum/__init__.py

```diff
@@ -1,3 +1,4 @@
-from ordered_enum.ordered_enum import OrderedEnum, ValueOrderedEnum  # noqa: F401
+from ordered_enum.ordered_enum import OrderedEnum as OrderedEnum
+from ordered_enum.ordered_enum import ValueOrderedEnum as ValueOrderedEnum
 
 __version__ = "0.0.8"
```

Comment:
Could you see if this works instead? 

```suggestion
__version__ = "0.0.8"

__all__ = [
    "__version__",
    "OrderedEnum",
    "ValueOrderedEnum",
]
```

If it does, then I think we don't need the `from y import X as X` pattern 🙂

Reply:
This should also work. Let me try it and put it up.

Reply:
Done! lmk if there's anything else.